core/test/integration/BaseFixture.cpp

```diff
@@ -94,6 +94,12 @@ api::AccountCreationInfo XRP_KEYS_INFO(
         {hex::toByteArray("f7e8d16154d3c7cbfa2cea35aa7a6ae0c429980892cf2d6ea9e031f57f22a63d")}
 );
 
+api::AccountCreationInfo VAULT_XRP_KEYS_INFO(
+        0, {"main"}, {"44'/144'/0'"},
+        {hex::toByteArray("03432A07E9AE9D557F160D9B1856F909E421B399E12673EEE0F4045F4F7BA151CF")},
```

Comment:
_All your keys are belong to us_.

Reply:
By the way, these are the test keys used for integration tests and QA on Vault.